protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```
@@ -655,6 +655,148 @@ definitions:
             }
         type: object
         existingJavaType: com.fasterxml.jackson.databind.JsonNode
+      oauth_connector_input_specification:
```

@bazarnov I realized we should  update this to include oauth2.0
```suggestion
      oauth_2_connector_input_specification:
```